xml/src/main/java/org/ehcache/xml/ConfigurationParser.java

```diff
@@ -339,7 +339,7 @@ public ListenersConfig listenersConfig() {
           }
 
           @Override
-          public Iterable<ResourcePool> resourcePools() {
+          public Collection<ResourcePool> resourcePools() {
```

Comment:
I have no problem with it but what prompted the change from `Iterable` to `Collection`?